src/style.css

```diff
@@ -1,3 +1,72 @@
+html{
+  background-color: wheat;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  size: 12px;
+}
+
+.header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+}
+
+.img{
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-left: 50px;
+}
+
+.nav{
+  font-size: 15px;
+  background-color: #ffff;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  text-transform: uppercase;
+}
+
+.nav_link{
+  padding: 25px;
+  text-decoration: none;
+  color: black;
+  height: 15px;
+  text-align: center;
+}
+
+.is_active::after{
+  content:'';
+  height: 4px;
+  display: inline-block;
+  background-color: #00ACDC;
+  border-radius: 8px;
+  width: 100%;
+}
+
+.nav_link:hover{
+  background-color: #00ACDC;
+}
+
+.link1{
+  color: #00ACDC;
+}
+
+.link1:hover{
+  background-color: black;
+}
+
+.link4{
+  color: #00ACDC;
+}
+
+.link4:hover{
+  background-color: black;;
```

Comment:
There is an extra semicolon at the end of the 'background-color' property. Remove the double semicolon to avoid potential issues.